features/aliases.feature

```diff
@@ -126,26 +126,115 @@ Feature: Create shortcuts to specific WordPress installs
     When I run `wp eval --skip-wordpress 'echo realpath( getenv( "RUN_DIR" ) );'`
     Then save STDOUT as {TEST_DIR}
 
-    When I run `wp cli alias`
+    When I run `wp cli alias list`
     Then STDOUT should be YAML containing:
       """
       @all: Run command against every registered alias.
       @foo:
         path: {TEST_DIR}/foo
       """
 
-    When I run `wp cli aliases`
+    When I run `wp cli alias list --format=json`
+    Then STDOUT should be JSON containing:
+      """
+      {"@all":"Run command against every registered alias.","@foo":{"path":"{TEST_DIR}/foo"}}
+      """
+
+  Scenario: Get alias information
+    Given an empty directory
+    And a config.yml file:
```

Comment:
@swissspidy config.yml is valid config filename but not in this location.l This can be fixed by changing 
```
When I run `wp cli alias get @foo`
```
to
```
When I run `WP_CLI_CONFIG_PATH=config.yml wp cli alias get @foo` 
```

Reply:
I see. In this case I would use `wp-cli.yml` though. Its not this test's job to verify whether `WP_CLI_CONFIG_PATH` + a custom file name works.

Reply:
We have a similar scenario here https://github.com/wp-cli/wp-cli/blob/a0c973927761ea84f79ac49c976931808f129e03/features/aliases.feature#L151

Reply:
@swissspidy I've updated config file name but doesn't seem to work as expected also checked in local setup but tests aren't passing, also [this](https://travis-ci.org/wp-cli/wp-cli/builds/428656307?utm_source=github_status&utm_medium=notification) build hasn't started yet 😕